### ISSUE TITLE
QUESTION: stream url for channel

### ISSUE THREAD

**Issue by aaamoeder** [add-ons/plugin.video.yelo#49]
I'm trying to find a stream url for one of the channels so I can then try to use it inside home assistant (automatically casting to tv) , but don't know where to look.
Anyone able to help ?

PS: sorry to post this here but didn't know where else to ask...

**Comment by fcools** [comment]
Well... that won't work for several reasons I think.
You need to authenticate yourself before you can start any stream, this could be done with a custom script (or addon) in HomeAssistant, but the main problem will be decoding the stream as I don't see a way to use or even install Widevine on HA.

My 2 cents.

**Comment by michaelarnauts** [comment]
Correct. You might look into starting an activity that uses the official Yelo app. A bit like asking the Google assistant to play a movie on Netflix.

**Comment by fcools** [comment]
Perhaps this could be of use: [https://github.com/skorokithakis/catt](url) and start it from HA.

**Comment by aaamoeder** [comment]
Thanks for your replies ! I hadn't thought it would be so complicated, guess I'll have to look for a different way.

**Comment by fripsy** [comment]
Is your goal having a card in HA displaying a TV stream?

**Comment by aaamoeder** [comment]
> Is your goal having a card in HA displaying a TV stream?

yes, but also being able to start a stream cast from an automation on a chromecast
